poetry/installation/pip_installer.py

```diff
@@ -189,7 +193,7 @@ def install_directory(self, package):
         else:
             req = os.path.realpath(package.source_url)
 
-        args = ["install", "--no-deps", "-U"]
+        args = ["install", "--no-deps", "-U", "--root", str(self._env.path)]
```

Comment:
I tried to trigger this condition because I think that `--root` might be wrong (the rest in this change correctly uses `--prefix`) but I could not figure out how to trigger `install_directory`. Even when I have:
```
django = {path = "django.git", develop = false}
```
it does not do a directory install. Do you have any tips for me here?

Reply:
@apollo13 can you try with latest changes please?